src/EnergyPlus/DataDaylighting.hh

```diff
@@ -335,25 +324,33 @@ struct DaylightingData : BaseGlobalStruct
 
     Array1D<DataDaylighting::EnclDaylightCalc> enclDaylight;
```

Comment:
Another thing I would suggest ... use `EPVector` rather than `Array1D` for all arrays of objects. `EPVector` is a much smaller template (i.e., it defines many fewer operators) and since the compiler has to compile every unique template in every compilation unit in full, this should speed up the build process.

Reply:
This is a huge suggestion for those of us that don't know.